test/e2e/config_test.go

```diff
@@ -313,6 +313,8 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: test-cm
+  annotations:
+    ann1: val1
```

Comment:
I've added this to show that not all annotations stripped when object is passed to rebase rules